commit/merkleroot/rmn/controller.go

```diff
@@ -17,7 +17,7 @@ import (
 
 	mapset "github.com/deckarep/golang-set/v2"
 	"golang.org/x/exp/maps"
-	rand2 "golang.org/x/exp/rand"
+	rand2 "golang.org/x/exp/rand" //nolint:staticcheck // ignoring deprecated golang.org/x/exp/rand rule for now.
```

Comment:
Not sure how this slipped through to go1.24 PR. I wasn't able to find a seed option in the suggested math/rand/v2 library.